utilities/utilities_setup.py

```diff
@@ -0,0 +1,14 @@
+import MultiPyVu as mpv
+from enum import Enum, auto
+
+class MVUInstrumentList(Enum):

```

Comment:
Let's please put this setup in a function so we can call it from our scripts.

Reply:
changed it to a function. Note the file name is also now changed to setup.py